packages/client/src/hooks/useNotifications.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  nextNotification,
+  evictNotification,
+} from "@/store/actions/notificationsActions";
+import { getActiveNotification } from "@/store/selectors/notifications";
+import { NotificationInterface } from "@/types/store";
+import { useEffect, useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+interface useNotificationsParams {
+  timeouts: {
+    minTimeout?: number;
+    maxTimeout?: number;
+  };
+}
+
+interface useNotifications {
+  active: NotificationInterface;
+  handleRemoveNotification: () => void;
+}
```

Comment:
1. This isn't used anywhere, aside from accidental export, a result of clashing of same named interface and a hook (as mentioned above ☝️ )
2. You can use and interface to declare a function, but it should be written like this:
```typescript
interface Function {
  (<params>): <return type>
}
```
and then you can just declare a function like so: `const function: Function = (params) => {/* ... */}`, without having to write the types for params and return again